<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Surefire suite" duration-ms="118886" started-at="2022-01-09T20:28:25 CET" finished-at="2022-01-09T20:30:24 CET">
    <groups>
    </groups>
    <test name="Surefire test" duration-ms="118886" started-at="2022-01-09T20:28:25 CET" finished-at="2022-01-09T20:30:24 CET">
      <class name="AppiumProject.JavaAppium.TextTest">
        <test-method status="PASS" signature="killAllNodes()[pri:0, instance:AppiumProject.JavaAppium.TextTest@1d8d30f7]" name="killAllNodes" is-config="true" duration-ms="3008" started-at="2022-01-09T20:28:34 CET" finished-at="2022-01-09T20:28:37 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killAllNodes -->
        <test-method status="FAIL" signature="Text(java.lang.String)[pri:0, instance:AppiumProject.JavaAppium.TextTest@1d8d30f7]" name="Text" duration-ms="1" started-at="2022-01-09T20:30:24 CET" data-provider="SwipeTestData" finished-at="2022-01-09T20:30:24 CET">
          <exception class="org.testng.internal.reflect.MethodMatcherException">
            <message>
              <![CDATA[
[public void AppiumProject.JavaAppium.TextTest.Text(java.lang.String) throws java.io.IOException,java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: Text([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 10,(java.lang.String) 25,(java.lang.String) 55]]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.reflect.MethodMatcherException: 
[public void AppiumProject.JavaAppium.TextTest.Text(java.lang.String) throws java.io.IOException,java.lang.InterruptedException] has no parameters defined but was found to be using a data provider (either explicitly specified or inherited from class level annotation).
Data provider mismatch
Method: Text([Parameter{index=0, type=java.lang.String, declaredAnnotations=[]}])
Arguments: [(java.lang.String) 10,(java.lang.String) 25,(java.lang.String) 55]
at org.testng.internal.reflect.DataProviderMethodMatcher.getConformingArguments(DataProviderMethodMatcher.java:43)
at org.testng.internal.Parameters.injectParameters(Parameters.java:914)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:34)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.reflect.MethodMatcherException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Text -->
      </class> <!-- AppiumProject.JavaAppium.TextTest -->
      <class name="AppiumProject.JavaAppium.SwipeTest">
        <test-method status="PASS" signature="killAllNodes()[pri:0, instance:AppiumProject.JavaAppium.SwipeTest@32d2fa64]" name="killAllNodes" is-config="true" duration-ms="3022" started-at="2022-01-09T20:28:31 CET" finished-at="2022-01-09T20:28:34 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killAllNodes -->
        <test-method status="PASS" signature="Swipe(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:AppiumProject.JavaAppium.SwipeTest@32d2fa64]" name="Swipe" duration-ms="30681" started-at="2022-01-09T20:29:24 CET" data-provider="SwipeTestData" finished-at="2022-01-09T20:29:55 CET">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[25]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[55]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Swipe -->
        <test-method status="PASS" signature="Swipe(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:AppiumProject.JavaAppium.SwipeTest@32d2fa64]" name="Swipe" duration-ms="29290" started-at="2022-01-09T20:29:55 CET" data-provider="SwipeTestData" finished-at="2022-01-09T20:30:24 CET">
          <params>
            <param index="0">
              <value>
                <![CDATA[10]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[20]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[35]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Swipe -->
      </class> <!-- AppiumProject.JavaAppium.SwipeTest -->
      <class name="AppiumProject.JavaAppium.ScrollTest">
        <test-method status="PASS" signature="killAllNodes()[pri:0, instance:AppiumProject.JavaAppium.ScrollTest@436e852b]" name="killAllNodes" is-config="true" duration-ms="3011" started-at="2022-01-09T20:28:28 CET" finished-at="2022-01-09T20:28:31 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killAllNodes -->
        <test-method status="PASS" signature="ScrollTest1()[pri:0, instance:AppiumProject.JavaAppium.ScrollTest@436e852b]" name="ScrollTest1" duration-ms="20351" started-at="2022-01-09T20:29:04 CET" finished-at="2022-01-09T20:29:24 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ScrollTest1 -->
      </class> <!-- AppiumProject.JavaAppium.ScrollTest -->
      <class name="AppiumProject.JavaAppium.ButtonTest">
        <test-method status="PASS" signature="killAllNodes()[pri:0, instance:AppiumProject.JavaAppium.ButtonTest@3234e239]" name="killAllNodes" is-config="true" duration-ms="3023" started-at="2022-01-09T20:28:25 CET" finished-at="2022-01-09T20:28:28 CET">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- killAllNodes -->
        <test-method status="FAIL" signature="ButtonTest1()[pri:0, instance:AppiumProject.JavaAppium.ButtonTest@3234e239]" name="ButtonTest1" duration-ms="26443" started-at="2022-01-09T20:28:37 CET" finished-at="2022-01-09T20:29:04 CET">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at AppiumProject.JavaAppium.ButtonTest.ButtonTest1(ButtonTest.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ButtonTest1 -->
      </class> <!-- AppiumProject.JavaAppium.ButtonTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
